test/currency.json

```diff
@@ -7,5 +7,12 @@
   [1.99999, "$0.00", "$2.00"],
   [1.99, "$0.00", "$1.99"],
   [1.99, "$0", "$2"],
-  [1000.40, "$0", "$1000"]
+  [1000.40, "$0", "$1000"],
+  [500, "#,##0\"€\";(#,##0\"€\")", "500€"],
+  [350, "$#,###.00;($#,###.00)", "$350.00"],
+  [-350, "$#,###.00;($#,###.00)", "($350.00)"],
+  [1777, "$#,###.00;($#,###.00)", "$1,777.00"],
+  [-1777, "$#,###.00;($#,###.00)", "($1,777.00)"],
+  [17777, "$#,###.00;($#,###.00)", "$17,777.00"],
+  [-17777, "$#,###.00;($#,###.00)", "($17,777.00)"]
```

Comment:
I would include another example that was a known error (and has more significant digits) like [these](https://github.com/looker/helltool/issues/32316#issuecomment-332295328)

`[-2084048.35, "$#,##0.00;($#,##0.00)", "($2,084,048.35)"]`

Reply:
:+1